gradle/projects.main.properties

```diff
@@ -20,6 +20,7 @@ nessie-quarkus-tests=servers/quarkus-tests
 nessie-rest-services=servers/rest-services
 nessie-s3minio=testing/s3minio
 nessie-s3mock=testing/s3mock
+nessie-multi-env-test-engine=testing/test-engine
```

Comment:
Can you rename the directory to `multi-env-test-engine`? "test-engine" is a bit too generic IMO.

Reply:
renamed